src/ui/server/helpers/page-variables/single-input/index.ts

```diff
@@ -19,13 +19,21 @@ const singleInputPageVariables = ({
   BACK_LINK,
   FEEDBACK_ROUTE = LINKS.EXTERNAL.FEEDBACK,
   ORIGINAL_URL,
+  HAS_SAVE_AND_BACK,
+  CUSTOM_CONTENT_HTML,
+  CONDITIONAL_YES_HTML,
+  CONDITIONAL_NO_HTML,
 }: SingleInputPageVariablesInput) => {
   const pageVariables: SingleInputPageVariables = {
     ...corePageVariables({
       PAGE_CONTENT_STRINGS,
       BACK_LINK,
       FEEDBACK_ROUTE,
       ORIGINAL_URL,
+      HAS_SAVE_AND_BACK,
+      CUSTOM_CONTENT_HTML,
+      CONDITIONAL_YES_HTML,
+      CONDITIONAL_NO_HTML,
```

Comment:
This is repeated for multiple page variable functions. Could we have a new shared helper?